app/views/rubygems/_gem_members.html.erb

```diff
@@ -1,28 +1,22 @@
 <div class="gem__members">
   <% if rubygem.owners.present? %>
     <h3 class="t-list__heading"><%= t '.owners_header' %>:</h3>
-    <div class="gem__users">
-      <%= links_to_owners(rubygem) %>
-    </div>
+    <ul class="tw-my-3">
+      <% show_mfa_status = policy(rubygem).show_owner_mfa_status? %>
+      <% rubygem.owners.sort_by(&:id).each do |owner| %>
```

Comment:
is the `sort_by` needed? shouldn't the association be ordered by id by default?

Reply:
By default they come out in the order of the ownership model:

```ruby
irb> Rubygem.find(211568).owners.map(&:id)
  Rubygem Load (0.9ms)  SELECT "rubygems".* FROM "rubygems" WHERE "rubygems"."id" = $1 LIMIT $2  [["id", 211568], ["LIMIT", 1]]
  User Load (1.3ms)  SELECT "users".* FROM "users" INNER JOIN "ownerships" ON "users"."id" = "ownerships"."user_id" WHERE "users"."deleted_at" IS NULL AND "ownerships"."rubygem_id" = $1 AND "ownerships"."confirmed_at" IS NOT NULL  [["rubygem_id", 211568]]
=> [2, 3, 6, 5, 4]
```